packages/core/src/awsService/appBuilder/models/metadata.json

```diff
@@ -0,0 +1,60 @@
+{
+    "patterns": {
+        "s3-lambda-resizing-sam": {
+            "name": "Resizing image",
+            "description": "Lambda, S3 • Python, Javascript, Java, .NET • SAM",
+            "runtimes": [
+                {
+                    "id": "python",
+                    "name": "Python"
+                },
+                {
+                    "id": "javascript",
+                    "name": "Javascript"
+                },
+                {
+                    "id": "dotnet",
+                    "name": "Dotnet"
+                },
+                {
+                    "id": "java",
+                    "name": "Java"
+                }
+            ],
+            "iac": [
+                {
+                    "id": "sam",
+                    "name": "SAM"
+                }
+            ]
```

Comment:
This will change from what we discussed.

Reply:
I'll add those changes in another PR